### PR TITLE
Deprecate pdk-module-template repo

### PR DESCRIPTION
**Pull request by bmjen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,3 @@
...
-
-
-### Gemfile
-
->A Gemfile is a file we create which is used for describing gem dependencies for Ruby programs. All modules should have an associated Gemfile for installing the relevant gems. As development and testing is somewhat consistant between modules we have used the template to define a set of gems relevant to these processes.
-
-| Key            | Description   |
-| :------------- |:--------------|
-|required|Allows you to specify gems that are required within the Gemfile. Gems can be defined here within groups, for example we use the :development gem group to add in several gems that are relevant to the development of any module.|
+This template repository will no longer work with [PDK v1.3.0](https://puppet.com/docs/pdk/1.3/pdk.html) and newer. Please point your fork to the new templates repository and perform a rebase. Thanks!
```
Comment (jbondpdx) [inline README.md]: Please change to present tense: "This template repository no longer works with PDK v.1.3.0 ..."

File: README.md [hunk 2]
```diff
@@ -1,68 +1,3 @@
-# PDK Module Template
+# pdk-module-template is deprecated in favor of [pdk-templates](https://github.com/puppetlabs/pdk-templates).
```
Comment (jbondpdx) [inline README.md]: If it doesn't work already, is "deprecated" correct? Maybe "has been replaced with" would be more accurate?

Comment (bmjen) [reply README.md]: Good point. I can update that.

**Comment by rodjek** [comment]
Updated with @jbondpdx's changes

**Pull request merged**